cmake/Templates/PTLConfig.cmake.in

```diff
@@ -1,10 +1,18 @@
 # -------------------------------------------------------------------------------------- #
+
+if("${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION}" LESS 2.8)
+   message(FATAL_ERROR "CMake >= 3.0.0 required")
+endif()
+if(CMAKE_VERSION VERSION_LESS "3.0.0")
```

Comment:
Why is this 3.0.0 and not 3.8.0?

Reply:
> The updated usage mirrors the CMake-generated Targets.cmake file: assertions to check for the rough CMake version, then a scoped policy change for the file itself.

I suspect cmake does this for better backward compatibility with old versions: each successive test requires newer features. If you look at the cmake-generated Targets from any installation you'll see the same assertions in the same order.

Reply:
Maybe not though... it's not clear in https://gitlab.kitware.com/cmake/cmake/-/blob/master/Source/cmExportCMakeConfigGenerator.cxx?ref_type=heads#L153-L186 where the required cmake version gets set. I'll update it to 3.8.